src/View/Draft.elm

```diff
@@ -125,6 +122,17 @@ draftView ({ draft, currentUser } as model) =
                 viewAutocompleteMenu model.draft
             else
                 text ""
+
+        ( hasSpoiler, charCount ) =
+            case draft.spoilerText of
+                Just spoilerText ->
+                    ( True, (String.length spoilerText) + draft.statusLength )
+
+                Nothing ->
+                    ( False, draft.statusLength )
+
+        limitExceeded =
+            charCount > 500
```

Comment:
There's sadly no way to access the server configuration for status max length, it seems https://github.com/tootsuite/documentation/blob/master/Using-the-API/API.md#instance

So for now this is an invariant, and is only used in this specific view. We may want to move it to the model the day we'll be able to fetch that information and update the model accordingly.